test/stdlib/Builtins.swift

```diff
@@ -99,6 +99,20 @@ tests.test("_isUnique_native/NonNativeTrap")
   expectCrashLater()
   _ = _isUnique_native(&x)
 }
+
+tests.test("_hasSideTable") {
+  var aS1 = X()
+  expectFalse(_hasSideTable(&aS1))
```

Comment:
@mikeash are there any other ways for objects to get side-tables besides gaining weak refs?

Reply:
Currently no, but I believe the side table is intended to expand to serve more needs in the future. For example, it would be the natural place to put stored properties declared in extensions, or other "associated object" like things.

Reply:
@mikeash what about actual associated objects? Isn't that going to require a side table entry?

Reply:
@mikeash Aren't side tables also generated in case of reference count overflows?